test.html

```diff
@@ -4,7 +4,5 @@
   </head>
   <body>
     <h1>Testing</h1>
```

Comment:
Another testing comments

- [ ] This could be an issue
- [ ] Another possibility issue